go.mod

```diff
@@ -5,15 +5,30 @@ go 1.19
 require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/turbot/go-kit v0.6.0
-	github.com/turbot/steampipe-plugin-sdk/v5 v5.5.0
-	google.golang.org/api v0.101.0
+	github.com/turbot/steampipe-plugin-sdk/v5 v5.5.1
```

Comment:
@madhushreeray30, Here we are updating the Steampipe SDK version. Do we have any breaking changes with this version?